exchange/utils.go

```diff
@@ -847,6 +859,49 @@ func mergeBidderRequests(allBidderRequests []BidderRequest, bidderNameToBidderRe
 	return allBidderRequests
 }
 
+func gdprFromGPP(r *openrtb2.BidRequest, gpp gpplib.GppContainer) {
+	if r.Regs != nil && r.Regs.GDPR == nil {
+		if len(r.Regs.GPPSID) > 0 {
```

Comment:
Should be `r.Regs.GPPSID != nil` to detect if SID exists. We can set GDPR=0 if `GPPSID` exists but is empty.

Reply:
Thanks, I forgot slices could be nil